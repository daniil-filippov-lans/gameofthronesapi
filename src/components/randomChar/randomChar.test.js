import React from 'react';
import RandomChar from './randomChar';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

describe('Testing <RandomChar/>', () => {
    it('RandomChar rendered correctly', () => {
        const char = shallow(<RandomChar />);
        expect(toJSON(char)).toMatchSnapshot();
    });

    // it('RandomChar state "char" is empty object', () => {
    //     const char = shallow(<RandomChar />);
    //     expect(char.state().char).toBeObject();
    // });
});
