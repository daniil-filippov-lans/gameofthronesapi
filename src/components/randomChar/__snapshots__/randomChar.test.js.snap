// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <RandomChar/> RandomChar rendered correctly 1`] = `
<div
  className="random-block rounded"
>
  <Spinner />
</div>
`;
